packages/wallets/src/features/cashier/modules/Transfer/provider/TransferProvider.tsx

```diff
@@ -19,19 +20,25 @@ export const useTransfer = () => {
     return context;
 };
 
-const TransferProvider = ({ children }: React.PropsWithChildren) => {
+type TProps = {
+    accounts?: THooks.TransferAccount[];
+};
+
+const TransferProvider: React.FC<React.PropsWithChildren<TProps>> = ({ accounts: transferAccounts, children }) => {
```

Comment:
`TransferProvider` component now can receive `accounts` from parent. If `accounts` exist TransferProvider component will not call `transfer_between_accounts` request again, because it was called previously in `WalletTransfer` flow component.
Since `Transfer` is a module, if someone wants to import this module, there is no need to pass `accounts`. In that case `accounts` will be fetched inside TransferProvider itself.

Reply:
Nice work man 🚀